static/css/sass/cdr-ui.scss

```diff
@@ -1000,6 +1006,10 @@ p.no-search-results {
   }
 }
 
+.item_container_count {
```

Comment:
Hmm, this rule is also empty, is it needed?

Reply:
Hmm, sorry thought I removed the empty rule. Maybe need to run static again to generate the updates